core/src/main/kotlin/maru/database/BeaconChain.kt

```diff
@@ -24,6 +24,17 @@ interface BeaconChain : AutoCloseable {
 
   fun getSealedBeaconBlock(beaconBlockNumber: ULong): SealedBeaconBlock?
 
+  fun getSealedBlocks(
+    startBlockNumber: ULong,
+    count: ULong,
+  ): List<SealedBeaconBlock> =
+    generateSequence(startBlockNumber) { it + 1UL }
+      .take(count.toInt())
```

Comment:
```suggestion
    (startBlockNumber..startBlockNumber + count)
```

Reply:
Tried that but it breaks returning no blocks when the count is zero. Could be fixed with some use of max but I think it ends cleaner this way keeping the take with use of max and min.